sematic/db/schema.sql.sqlite

```diff
@@ -149,9 +146,13 @@ INSERT INTO "schema_migrations" (version) VALUES
   ('20221212110018'),
   ('20221214142609'),
   ('20221215212459'),
+  ('20221219145906'),
   ('20230105163857'),
   ('20230105170402'),
   ('20230217235835'),
   ('20230221004028'),
   ('20230221004041'),
-  ('20230222013045');
+  ('20230222013045'),
+  ('20230223020405'),
+  ('20230307230950'),
+  ('20230314141123');
```

Comment:
Why so many new migrations? Did we forget to add some in prior PRs?

Reply:
it appears so